var/spack/repos/builtin/packages/rocm-openmp-extras/package.py

```diff
@@ -47,7 +53,9 @@
     "f3e19699ce4ac404f41ffe08ef4546e31e2e741d8deb403b5477659e054275d5",
     "f41f661425534b5cfb20e2c0efd9d0800609dc3876ee9c3f76f026d36abbfa35",
     "d6c3f3aaa289251a433d99d1cffe432812093089ae876a6863295a15066c1eaf",
-    "13d3525078fd1c569f7c8ea7fce439b04f6b03814bbe88600c08f95c788e7802"
+    "13d3525078fd1c569f7c8ea7fce439b04f6b03814bbe88600c08f95c788e7802",
+    "13d3525078fd1c569f7c8ea7fce439b04f6b03814bbe88600c08f95c788e7802",
```

Comment:
Is this correct that shas are the same?

Reply:
Yes, the sha (13d3525) stayed the same for 4.3.0 to 4.3.1. 4.5.0 updated to 3d7277fd6.